.travis.yml

```diff
@@ -7,3 +7,6 @@ rvm:
   - 2.1.4
   - 2.2.0
   - 2.2.1
+before_install:
+- gem install bundler
+- export TZ=Australia/Canberra
```

Comment:
Yes. It's so the local_time tests are in somewhere other than UTC or else they fail.